laboratory/src/org/labkey/laboratory/LaboratoryModule.java

```diff
@@ -212,11 +214,11 @@ protected void registerSchemas()
 
     @NotNull
     @Override
-    public LinkedHashSet<ClientDependency> getClientDependencies(Container c)
+    public List<Supplier<ClientDependency>> getClientDependencies(Container c)
     {
         // allow other modules to register with EHR service, and include their dependencies automatically
         // whenever laboratory context is requested
-        LinkedHashSet<ClientDependency> ret = new LinkedHashSet<>();
+        List<Supplier<ClientDependency>> ret = new LinkedList<>();
```

Comment:
any reason for LinkedList vs ArrayList?

Reply:
Less memory overhead. I prefer them in cases like this where we're just adding and iterating in order, where random access isn't used.